CMakeLists.txt

```diff
@@ -307,7 +307,14 @@ endif(WITH_QATZIP)
 
 # needs mds and? XXX
 option(WITH_LIBCEPHFS "libcephfs client library" ON)
-option(WITH_LIBCEPHSQLITE "libcephsqlite client library" ON)
+
+find_package(SQLite3)
+if(SQLITE3_FOUND)
+  option(WITH_LIBCEPHSQLITE "libcephsqlite client library" ON)
```

Comment:
You still want this to default to ON?

Reply:
This won't be merged as-is for now. It's just a dumping ground for patches. Leaving it `ON` just makes building this easier.